### DISCUSSION TITLE
HARVESTER is STOP running

### DISCUSSION THREAD

**Discussion by Mlalosevic** [Chia-Network/chia-blockchain#5286] [Support]
I have the issue with a harvester which occasionally stops. The state of process becomes **D (uninterruptible sleep)**

3164 pts/0 Dl 122:29 chia_harvester

checking the debug log does not indicate any issue. The process cannot be restarted (-r). Further to that, "kill -9" is not working either. This is frustrating because it does not have exact pattern when it can happen, once in three days or in one hour.  The spindown and sleep I have disabled using hdparam on all disks in farm.

Gui is not indicating such situation (when I was farming using GUI). It can be indicated that you are farming while in fact the harvester has stopped. Therefore, I start farming from terminal and grep command is giving me the following information when harvester is stopped:    

2951 pts/0 SLl 1:02 chia_daemon

3164 pts/0 Dl 122:29 chia_harvester

3165 pts/0 SLl 6:31 chia_farmer

3166 pts/0 Sl 606:12 chia_full_node

3167 pts/0 SLl 102:55 chia_wallet

3936 pts/0 S 6:05 chia_full_node

3941 pts/0 R 25:10 chia_full_node

3942 pts/0 S 25:08 chia_full_node

3943 pts/0 S 25:10 chia_full_node

3944 pts/0 S 25:13 chia_full_node

72999 pts/2 S+ 1:12 watch -n 5 chia farm summary

If somebody can give me any clue it would be appreciated. Manny thx

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
